File: Java Script - Introduction to Programming/4.1. Complex Conditions/11. Cinema.js
function cinema([premiere, normal , discount]) {
  let chairs = normal * discount

  switch (premiere) {
    case 'Premiere': chairs *= 12.00; break
    case 'Normal':   chairs *= 7.50; break
    case 'Discount': chairs *= 5.00; break
  }
  
  return `${chairs.toFixed(2)} leva`
}

//console.log(cinema(['Normal', ' 21', '13']))

File: Pyrhon - Introduction to Programming/4.1. Complex Conditions/11. Cinema.py
premiere, normal , discount = [input() for _ in range(3)]
chairs = float(normal) * float(discount)

if premiere == 'Premiere':
    chairs *= 12.00
elif premiere == 'Normal':
    chairs *= 7.50
elif premiere == 'Discount':
    chairs *= 5.00

print(round(chairs, 2), 'leva')


'''
Premiere
10
12
#1440.00 leva
'''

